### PR TITLE
unit_of_measurement support

### PR DESCRIPTION
**Pull request by ALERTua**
optional unit_of_measurement parameter
Before:
image
After:
image

### CODE REVIEW

File: src/utils/getHomassistantDiscoveryJson.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { getMQTTTopic } from "./getMQTTTopic";
 
 const getJsonObject = (
   key: DataKey,
-  device_class: "temperature" | "humidity" | "carbon_dioxide" | "battery"
+  device_class: "temperature" | "humidity" | "carbon_dioxide" | "battery",
+  unit_of_measurement?: string
```
Comment (niklasarnitz) [inline src/utils/getHomassistantDiscoveryJson.ts]:
```suggestion
  unit_of_measurement?: "°C" | "%" | "ppm"
```

**Review by niklasarnitz — APPROVED** [review]
The one small change to include some better typings should be all :)

The rest lgtm.

**Comment by ALERTua** [comment]
Thank you. I have zero knowledge of TypeScript, it was all Google Bard :)

**Comment by niklasarnitz** [comment]
No worries :)

**Pull request merged**